mitodl/bootcamp-ecommerce
# Refresh old interview links

#### What are the relevant tickets?
Closes #958 

#### What's this PR do?
Updates jobma interview links that satisfy certain criteria:
- "pending" submission status
- valid application state(not approved, rejected, awaiting payment, or refunded)
- older than X days (where X is determined by `JOBMA_LINK_EXPIRATION_DAYS`, default 29)
- bootcamp run start date is not in the past

#### How should this be manually tested?
- If you already have some existing pending interview submissions that match the above criteria, run the task manually and verify that the old interview is deleted and a new one is created, with a working jobma link.
- If you don't have existing interview submissions, create 1 or more.  Set `JOBMA_LINK_EXPIRATION_DAYS` to 0 in your .env file, then manually run the task.
